### PR TITLE
Checks for chatroom on load of map

### PR DESCRIPTION
**Pull request by wayneadams**
resolves #23

### CODE REVIEW

File: src/map/map.js [hunk 1]
```diff
@@ -33,21 +33,40 @@ export default class Map extends Component {
...
     };
   }
 
   componentDidMount() {
-    this.watchID = navigator.geolocation.watchPosition((position) => {
+    this.listenForGeoChange();
+  }
+
+  componentWillUnmount() {
+    navigator.geolocation.clearWatch(this.watchID);
```
Comment (hchsquared) [inline src/map/map.js]: Where is the Watch set on watchID now that it is no longer in componentDidMount?

File: src/map/map.js [hunk 2]
```diff
@@ -33,21 +33,40 @@ export default class Map extends Component {
...
+      }
+    });
+  }
+
+  checkForChatRoom() {
+    this.props.socket.emit('updateMessagesState', this.state.coords);
+    this.props.socket.on('updateMessagesState', result => {
+      log('result of updateMessagesState', result);
+      if (result === null) {
+        this.setState({ newRoom: true });
```
Comment (hannahhenderson) [inline src/map/map.js:65]: Do you need to call createNewChatRoom() here?

**Comment by hannahhenderson** [comment]
See comment, otherwise lgtm

File: src/map/map.js (hunk 2 above)
Comment (wayneadams) [inline src/map/map.js:65]: it's a placeholder function for future functionality

**Pull request merged**